src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/speed/modes/intave/SpeedIntave14.kt

```diff
@@ -137,17 +130,28 @@ class SpeedIntave14(override val parent: ChoiceConfigurable<*>) : SpeedBHopBase(
                 mc.options.rightKey.isPressed -> -90f
                 else -> 0f
             }
+        } else {
+            0f
         }
         return 0f
     }
 
     private fun airYawOffset(): Float {
-        return if (!player.isOnGround &&
-            mc.options.forwardKey.isPressed && !mc.options.leftKey.isPressed && !mc.options.rightKey.isPressed
-        ) {
-            -45f
-        } else {
-            0f
+        yaw = when {
+            !player.isOnGround &&
+                mc.options.forwardKey.isPressed &&
+                !mc.options.leftKey.isPressed &&
+                !mc.options.rightKey.isPressed
+                -> -45f
+
+            else -> 0f
```

Comment:
what is this formatting?

Reply:
since I need to move to a new line at least once to be under the max line length might as well just move every && down